recipes/symfem/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "symfem" %}
+{% set version = "2021.7.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 610aa8a3fc006c78a8e31e6c6cba270a70f6d0e8460b3920de0de326805b831d
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - numpy
+    - python >=3.6
+    - pytest
```

Comment:
It doesn't look like `pytest` is actually needed to use the library?

Reply:
It isn't. I've remove it now